srcpkgs/paraview/template

```diff
@@ -0,0 +1,62 @@
+# Template file for 'paraview'
+pkgname=paraview
+version=5.6.1
+revision=1
+wrksrc=ParaView-v${version}
+build_style=cmake
+configure_args="-D_GNU_SOURCE
```

Comment:
Nope. Remove it from configure_args. I meant to add it to the template like this (normally somewhere between checksum and post-Install):
```bash
CFLAGS="-D_GNU_SOURCE"
CXXFLAGS="-D_GNU_SOURCE"
```

Reply:
Ah, gotcha. Changed accordingly in 99d17dac47.